### PR TITLE
update version of appengine-gcs-client

### PR DESCRIPTION
**Pull request by elharo**
@frankyn @ludoch Let's get this internally synced up in preparation for deprecating and archiving the whole project.

### CODE REVIEW

File: java/example/pom.xml [hunk 1]
```diff
@@ -11,14 +11,14 @@
     <properties>
         <app.id>com.google.appengine.examples.gcs-appengine</app.id>
         <app.version>1</app.version>
-        <appengine.version>1.9.36</appengine.version>
+        <appengine.version>1.9.64</appengine.version>
```
Comment (frankyn) [inline java/example/pom.xml]: is an internal version of this library usually updated outside of updating this repo?

Comment (elharo) [reply java/example/pom.xml]: I'm not sure I follow. Can you elaborate? Do you mean something in google3?

This PR is purely about updating the example code.

Comment (frankyn) [reply java/example/pom.xml]: I misread the path, thanks for clarifying.

**Comment by elharo** [comment]
Thanks. Someone with write access will have to merge this.

**Pull request merged**